dev/com.ibm.ws.wssecurity_fat.wsscxf/fat/src/com/ibm/ws/wssecurity/fat/cxf/FATSuite.java

```diff
@@ -72,14 +72,8 @@
 
 public class FATSuite {
 
-    //The following repeats run Lite only for local testing
-    //@ClassRule
-    //public static RepeatTests r = RepeatTests.withoutModification().andWith(FeatureReplacementAction.EE8_FEATURES().removeFeature("jsp-2.2").removeFeature("jaxws-2.2").removeFeature("servlet-3.1").removeFeature("usr:wsseccbh-1.0").addFeature("jsp-2.3").addFeature("jaxws-2.3").addFeature("servlet-4.0").addFeature("usr:wsseccbh-2.0"));
-
-    //The following repeats run both Full and Lite buckets.
-    //To run Full only single test class locally,  using the command option "-Dfat.test.class.name";
-    //To run Full only entire bucket locally, comment out the Lite bucket above
+    //The following run EE7 and EE8 full fat and EE9 lite fat
     @ClassRule
-    public static RepeatTests r = RepeatTests.with(new EmptyAction().fullFATOnly()).andWith(FeatureReplacementAction.EE8_FEATURES().removeFeature("jsp-2.2").removeFeature("jaxws-2.2").removeFeature("servlet-3.1").removeFeature("usr:wsseccbh-1.0").addFeature("jsp-2.3").addFeature("jaxws-2.3").addFeature("servlet-4.0").addFeature("usr:wsseccbh-2.0"));
+    public static RepeatTests r = RepeatTests.with(new EmptyAction().fullFATOnly()).andWith(FeatureReplacementAction.EE8_FEATURES().fullFATOnly().removeFeature("jsp-2.2").removeFeature("jaxws-2.2").removeFeature("servlet-3.1").removeFeature("usr:wsseccbh-1.0").removeFeature("appSecurity-2.0").removeFeature("jsp-2.2").removeFeature("jaxws-2.2").removeFeature("servlet-3.1").removeFeature("usr:wsseccbh-1.0").addFeature("jaxws-2.3").addFeature("servlet-4.0").addFeature("usr:wsseccbh-2.0")).andWith(FeatureReplacementAction.EE9_FEATURES().removeFeature("jsp-2.3").removeFeature("jaxws-2.3").removeFeature("servlet-4.0").removeFeature("appSecurity-2.0").addFeature("appSecurity-4.0").addFeature("pages-3.0").addFeature("xmlWS-3.0").addFeature("servlet-5.0"));
```

Comment:
looks like some of the removeFeature calls are duplicated for ee9 repeat.

also, i think u need to add `addFeature("appSecurity-3.0")` and `addFeature("jsp-2.3")` at the end of ee8 repeat.

consequently, `removeFeature("appSecurity-2.0")` should probably be `removeFeature("appSecurity-3.0")` for ee9 repeat.

```
public static RepeatTests r = 

RepeatTests.with(new EmptyAction().fullFATOnly())

.andWith(FeatureReplacementAction.EE8_FEATURES().fullFATOnly()
.removeFeature("jsp-2.2")
.removeFeature("jaxws-2.2")
.removeFeature("servlet-3.1")
.removeFeature("usr:wsseccbh-1.0")
.removeFeature("appSecurity-2.0")
.removeFeature("jsp-2.2")
.removeFeature("jaxws-2.2")
.removeFeature("servlet-3.1")
.removeFeature("usr:wsseccbh-1.0")
.addFeature("jaxws-2.3")
.addFeature("servlet-4.0")
.addFeature("usr:wsseccbh-2.0"))

.andWith(FeatureReplacementAction.EE9_FEATURES()
.removeFeature("jsp-2.3")
.removeFeature("jaxws-2.3")
.removeFeature("servlet-4.0")
.removeFeature("appSecurity-2.0")
.addFeature("appSecurity-4.0")
.addFeature("pages-3.0")
.addFeature("xmlWS-3.0")
.addFeature("servlet-5.0"));
```

Reply:
Thanks for catching it.  Just noticed few redundant in my original line.  Luckily the liberty runtime able to get the expected feature dependency installed.   But I think `appSecurity` probably won't be needed to call out explicitly.  This is the update I'll be making:

public static RepeatTests r = RepeatTests.with(new EmptyAction().fullFATOnly()).andWith(FeatureReplacementAction.EE8_FEATURES().fullFATOnly().removeFeature("jsp-2.2").removeFeature("jaxws-2.2").removeFeature("servlet-3.1").removeFeature("usr:wsseccbh-1.0").addFeature("jsp-2.3").addFeature("jaxws-2.3").addFeature("servlet-4.0").addFeature("usr:wsseccbh-2.0")).andWith(FeatureReplacementAction.EE9_FEATURES().removeFeature("jsp-2.3").removeFeature("jaxws-2.3").removeFeature("servlet-4.0").addFeature("pages-3.0").addFeature("xmlWS-3.0").addFeature("servlet-5.0"));